.github/workflows/sdk_protos_map.csv

```diff
@@ -248,9 +248,14 @@ navigation,Close,close,Close,
 
 ## SLAM
 slam,GetPosition,get_position,Position,getPosition
-slam,GetPointCloudMap,get_point_cloud_map,PointCloudMap,getPointCloudMap
-slam,GetInternalState,get_internal_state,InternalState,getInternalState
+## HACK: SLAM (Go) implements proto GetPointCloudMap in user-facing helper PointCloudMapFull instead:
+slam,GetPointCloudMap,get_point_cloud_map,,getPointCloudMap
+## HACK: SLAM (Go) implements proto GetInternalState in user-facing helper InternalStateFull instead:
+slam,GetInternalState,get_internal_state,,getInternalState
 slam,GetProperties,get_properties,Properties,getProperties
+## HACK: SLAM (Go) provides 2 additional helper functions, adding 2 pseudo-entries:
+slam,InternalStateFull,,InternalStateFull,
```

Comment:
IDK how the comma syntax works but I assume you know what you're doing :D

Reply:
Double commas in the middle omits the Go method entirely (equiv to '' string)! Each row must have 4 commas.